tests/test_builder_containerbuild.py

```diff
@@ -383,11 +384,18 @@ def _mock_osbs(self, koji_build_id, src, koji_task_id, orchestrator=False, sourc
         osbs = flexmock()
 
         if orchestrator:
-            (osbs
-                .should_receive('create_orchestrator_build')
-                .with_args(koji_task_id=koji_task_id, **create_build_args)
-                .times(0 if build_not_started else 1)
-                .and_return(build_response))
+            if with_osbsvalidationexception:
+                (osbs
+                    .should_receive('create_orchestrator_build')
+                    .with_args(koji_task_id=koji_task_id, **create_build_args)
+                    .times(1)
+                    .and_raise(builder_containerbuild.ContainerError))
```

Comment:
Why this raises `ContainerError` and not `OsbsException`?

Reply:
IIUC, that's how it works... koji-c recieves OsbsValidationException and turns it into ContainerError

Reply:
create_orchestrator_build raises OsbsValidationException
and koji-c containerBuild raises then ContainerError

current setup isn't testing any change, because you basically with this bypass all code you've wrote

Reply:
:astonished: